website/static/js/filesWidget.js

```diff
@@ -23,7 +23,6 @@ var FilesWidget = oop.defclass({
             filesData: filesUrl,
             uploads: true,
             showFilter: true,
```

Comment:
Oh, no that doesn't break the width. It actually doesn't do anything at all (according to Caner) so I removed it.